client/src/feature/Login/Login.jsx

```diff
@@ -70,29 +66,80 @@ const Login = () => {
     setPassword(password);
   };
 
+  const handleLoginLinkClick = () => {
+    navigate("/signup");
+  };
+
   // 등록된 계정이 아닐경우 status가 404면 존재하지 않는 회원입니다..?
-    // 그럼 만약 비밀번호만 틀린경우 status가 다르게 뜨는지?
+  // 그럼 만약 비밀번호만 틀린경우 status가 다르게 뜨는지?
 
   return (
-    <div>
-      <input
-        type="email"
-        placeholder="Email"
-        value={email}
-        onChange={handleEmailValidation}
-      />
-      {validation.email && <p>{validation.email}</p>}
-      <br/>
-      <input
-        type="password"
-        placeholder="Password"
-        value={password}
-        onChange={handlePasswordValidation}
+    <>
+      <InputBox>
+        Email:
+        <Input
+          size={"32.4rem"}
+          height={"4.8rem"}
+          fontSize={"1rem"}
+          type="email"
+          placeholder="Email"
+          value={email}
+          onChange={handleEmailValidation}
+        />
+        {validation.email && <ValidMsg>{validation.email}</ValidMsg>}
+      </InputBox>
```

Comment:
```suggestion
      <InputBox>
        Email:
        <Input
          size={"32.4rem"}
          height={"4.8rem"}
          fontSize={"1rem"}
          type="email"
          placeholder="Email"
          value={email}
          onChange={handleEmailValidation}
          info={validation.email}
        />
      </InputBox>
```

이렇게 하면 되지 않나용? 저는 프로필 페이지에서 validation 저렇게 사용했습니당

Reply:
Input 사용법이 정확히 안나와 있어서 한번더 여쭤보고 수정 예정입니다!